cmd/kube-apiserver/app/options/options.go

```diff
@@ -76,6 +76,8 @@ type ServerRunOptions struct {
 	ProxyClientCertFile string
 	ProxyClientKeyFile  string
 
+	PeerCAFile string
```

Comment:
Are all three of these flags for the east-west (peer to peer) traffic?  Would it make sense for the flag names to share a common prefix (e.g. "peer-")?

Reply:
ProxyClient* files are for requestheader auth done by kube-aggregator to open a secure channel with the Aggregated API servers. 

The PeerCA file is for different kube-aggregators/kube-apiservers to verify each other. 

I guess we should restrict "peer" to be used just for communication between different kube-apiservers?

Reply:
Ah, thanks for explaining.  Yes, I'd expect "peer" to be used exclusively for communication between kube-apiservers, I was also expecting to have certs and keys that are exclusive for that direction of traffic.

Reply:
Actually, thanks for the reminder for using a separate set of certs for the purpose of UVIP.

Here is the current setting: 

1. We are (also) using ProxyClient* files to do client auth in UVIP i.e.  the server(destination apiserver) verifies the client (source apiserver)  using ProxyClient* certs

2. And the PeerCA is used by the client (source apiserver) to verify its talking to the right server (destination server)

So there's some overlap for ProxyClient* certs here that are serving a dual purpose:
a. Used for verification of Aggregated apiservers
b. Used for verification of remote apiservers

I am open to suggestions for using some prefix for these files that would make sense for this setting..